README.md

```diff
@@ -185,12 +185,23 @@ The user can also press `esc` to toggle the ability cycle through the options wi
 By default, the select prompt is limited to showing 7 options at a time
 and will paginate lists of options longer than that. This can be changed a number of ways:
 
-```golang
-// as a field on a single select
-prompt := &survey.MultiSelect{..., PageSize: 10}
+#### Select option's description
 
-// or as an option to Ask or AskOne
-survey.AskOne(prompt, &days, survey.WithPageSize(10))
```

Comment:
Why does this remove the WithPageSize example? Can the previous documentation also stay?

Reply:
Sorry, no idea why I did that